cmake/FindDAGMC.cmake

```diff
@@ -10,7 +10,7 @@
 message(STATUS ${DAGMC_ROOT})
 
 find_path(DAGMC_CMAKE_CONFIG NAMES DAGMCConfig.cmake
-          HINTS ${DAGMC_ROOT}
+          HINTS ${DAGMC_ROOT} $ENV{DAGMC_ROOT}
```

Comment:
Does this work? I thought the correct syntax was `HINTS ${DAGMC_ROOT} ENV DAGMC_ROOT`

Reply:
Just tested it out with the OpenMC project and it seems to work. 

Documentation on this `ENV` is [here](https://cmake.org/cmake/help/v3.0/variable/ENV.html). The syntax you included may be equivalent though, I'm not sure.

Reply:
I was going off of what it says [here](https://cmake.org/cmake/help/v3.8/command/find_path.html). Regardless, I guess they do the same thing.